lib/charms/vault_k8s/v0/vault_client.py

```diff
@@ -469,6 +482,18 @@ def is_common_name_allowed_in_pki_role(self, role: str, mount: str, common_name:
             logger.warning("Role does not exist on the specified path.")
             return False
 
+    def get_role_max_ttl(self, role: str, mount: str) -> Optional[int]:
```

Comment:
In the method above, the max_ttl was passed in as a string (ex. "120m"), when we run a get now we get an int (?). Is that really the case and if yes, what does it represent? hours?

Reply:
Reading charm.py it seems like it is seconds. Let's make sure that's correct and let's document this in the method docstrings.

Reply:
Yes, this is a weird one, but that's what the API returns.
Docstring updated